### ISSUE TITLE
Alert in nw.js 0.13.2

### ISSUE THREAD

**Issue by joelz** [nwjs/nw.js#4666]
I'm using nw.js v0.13.2  on win ia32 sdk on Windows 7 sp1 x64。

When using the `global.alert()`, I got something like this:

![image](https://cloud.githubusercontent.com/assets/469974/14202067/7d82a69c-f828-11e5-973c-9dffbf0ec82c.png)

Any clues why this red bold border appears ? Is it a  bug or I missed some setup ?

Thanks.

**Comment by VoidVolker** [comment]
Just tested: no red borders. Looks like it is your local problem.

**Comment by joelz** [comment]
Thanks for your test, @VoidVolker .

I tested this in another windows 7 x86 and a windows 10 x64, the result is the same. 

The app I used for all those tests is packaged using nw-builder v2.2.0.   

I also tried this: 
I package my app in the way described in the wiki page (firstly zip all necessary files into a file; then rename the file extension from .zip to .nw.; after that , `copy /b nw.exe+app.nw app.exe` to get the final app.exe) . And with this app.exe, the alert dialog shows normally: these red borders are gone.

So there may be something wrong during the nw-builder v2.2.0 package process. I'll open a issue in nw-builder repo, and this issue can be closed.

Thanks to all of you guys for your help.
